guide/xml/portfile-phase.xml

```diff
@@ -1044,7 +1044,8 @@ svn.revision        2100</programlisting>
         <listitem>
           <para>Checksum(s) of the distribution files. For ports with multiple
           distribution files, filenames must be included to associate files
-          with their checksums.</para>
+          with their checksums.  Each checksum entry must also indicate the
+          file's size.</para>
```

Comment:
I would go with "should" here, as below, instead of "must".

Reply:
Fixed.